.github/workflows/build.yml

```diff
@@ -3,7 +3,7 @@ name: Build documentation pages
 on:
   push:
     branches:
-      - '*'
+    - '*'
```

Comment:
indentation change not needed, but also has no effect, Github has both indentation styles in the guide: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths